internal/api/schemas/v1.yaml

```diff
@@ -21,7 +21,86 @@ paths:
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/GatewayList"
+                "$ref": "#/components/schemas/GatewayPage"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Error"
+    post:
+      description: 'Creates a new gateway.'
+      operationId: createGateway
+      operationId: addGateway
+      requestBody:
+        description: Gateway to add
```

Comment:
be consistent w/ "add" vs. "create". I prefer "create" fwiw

Reply:
yeah, meant to use `createGateway` -- I'll clean up the dup